src/app/shared/components/template/components/button/button.component.ts

```diff
@@ -37,6 +37,8 @@ interface IButtonParams {
   iconSecondary: string;
   /** TEMPLATE PARAMETER: "image_asset". The path to an image asset */
   image: string;
+  /** TEMPLATE PARAMETER: "button_icon_align". ALigns the icon to the left or right */
```

Comment:
nit (non-blocking): Would be good to specify that this is the _primary_ icon that gets aligned, just to be explicit